nautobot/docs/development/jobs/index.md

```diff
@@ -799,11 +798,11 @@ class NewBranch(Job):
         description="Access switch model", model=DeviceType, query_params={"manufacturer_id": "$manufacturer"}
     )
 
-    def run(self, location_name, switch_count, switch_model):
+    def run(self, location_name, switch_count, switch_model, manufacturer):
```

Comment:
While you're touching this, technically these are all keyword arguments, and manufacturer is optional, so this would be even more correct:

```suggestion
    def run(self, *, location_name, switch_count, switch_model, manufacturer=None):
```

Reply:
Added suggestion manually.